stake-o-matic/src/main.rs

```diff
@@ -372,7 +372,7 @@ fn classify_block_producers(
             }
         );
 
-        confirmed_blocks.push(retry_rpc_operation(10, || {
+        confirmed_blocks.push(retry_rpc_operation(42, || {
```

Comment:
:slot_machine:

Reply:
It didn't help :(

Reply:
I managed to clear a whole epoch with the step reduced to `2_500`.  Super slow though